index.html

```diff
@@ -28,14 +28,6 @@
             </ul>
         </nav>
     </header>
-    <section id="menu" class="menu-invisible">
-        <button type="button" id="closeButton"><img src="icons/close-icon.svg" alt="close icon"></button>
-        <ul class="container content menu">
-            <li class="textPlaceholder menu-item"><a href="#projects" aria-label="Portfolio"> Portfolio</a></li>
-            <li class="textPlaceholder menu-item"><a href="#about" aria-label="About"> About</a></li>
-            <li class="textPlaceholder menu-item"><a href="#contact" aria-label="Contact"> Contact</a></li>
```

Comment:
- [ ] Please ensure that you phrase your comments politely, as suggestions instead of orders. For example:

- This is a polite suggestion: "I think it would be better not to leave unused code"
- This reads like an order: "do not leave unused code"

Reply:
I think it would be better not to leave unused code in the codebase.